meta/main.yml

```
@@ -2,10 +2,10 @@
 galaxy_info:
   author: Olivier Pouilly
   role_name: mysql-replication-master-slave
```

Didn't catch this one on the first two readings. Dash is not a valid character in a role name for Galaxy.

```suggestion
  role_name: mysql_replication
```